두-정수사이의-합.js

```diff
@@ -0,0 +1,14 @@
+function solution(a, b) {
+    let answer = 0;
+    if (a - b >= 0) {
+        for (var i = b; i <= a; i++) {
+            answer += i;
+        }
+    } else {
+        for (var j = a; j <= b; j++) {
```

Comment:
```suggestion
    let answer = 0;
    
    if (a - b >= 0) {
        for (let i = b; i <= a; i++) {
            answer += i;
        }
    } else {
        for (let j = a; j <= b; j++) {
```
var는 꼭 필요한 경우가 아니라면 사용을 자제해주시고 let을 사용해주세요.